integration/extension-import/base.ts

```diff
@@ -0,0 +1,82 @@
+// Code generated by protoc-gen-ts_proto. DO NOT EDIT.
```

Comment:
This and other related files in this folder we generated from the existing `base.proto` as part of performing a build for tests.